cluster/examples/kubernetes/nfs/nfs-ceph.yaml

```diff
@@ -3,6 +3,12 @@ kind: Namespace
 metadata:
   name:  rook-nfs
 ---
+apiVersion: v1
+kind: ServiceAccount

```

Comment:
A service account will typically need cluster bindings or cluster role bindings to grant privileges. In this case, does the service account only require the SCC privs?

Reply:
aright, this service account is only for scc.